pkg/chunk/dynamodb_table_client.go

```diff
@@ -192,15 +196,18 @@ func (d dynamoTableClient) UpdateTable(ctx context.Context, current, expected Ta
 			return err
 		}
 
-		return d.backoffAndRetry(ctx, func(ctx context.Context) error {
-			return instrument.TimeRequestHistogram(ctx, "DynamoDB.TagResource", dynamoRequestDuration, func(ctx context.Context) error {
-				_, err := d.DynamoDB.TagResourceWithContext(ctx, &dynamodb.TagResourceInput{
-					ResourceArn: tableARN,
-					Tags:        expected.Tags.AWSTags(),
+		tags := expected.Tags.AWSTags()
```

Comment:
Yes if you remove the flag then this will try to untag them and fail.  We'll need to add Untag support at some point.  But its not important now.